docs/src/components/CodeBlock.tsx

```diff
@@ -71,8 +71,8 @@ export function Pre({ className, ...props }: React.ComponentProps<'pre'>) {
       className="CodeBlockPreContainer"
       onKeyDown={(event) => {
         if (
-          event.key === 'a' &&
-          (event.metaKey || event.ctrlKey) &&
+          (event.ctrlKey || event.metaKey) &&
+          String.fromCharCode(event.keyCode) === 'A' &&
```

Comment:
The logic was wrong; it ignored, e.g. Dvorak keyboard. This is a straight copy & paste from Material UI, not related to the PR, but I saw it, so why not? For those who wonder, at the root, it's an issue with the web standard.

Reply:
Is it expected that we are replacing a suggested usage (`KeyboardEvent.key`) with a [deprecated one](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)? 🤔